test/invites.test.js

```diff
@@ -46,15 +46,12 @@ test('lists correct group invite and accepting actually does something', async (
   t.pass('alice and bob replicate')
 
   const invites = await pull(bob.tribes2.listInvites(), pull.collectAsPromise())
+    .catch(t.error)
   t.equal(invites.length, 1, 'bob has 1 invite')
 
   const invite = invites[0]
   t.equal(invite.id, group.id, 'correct group id in invite')
-  t.true(invite.writeKey.key.equals(group.writeKey.key), 'correct writeKey')
-  t.true(
-    invite.readKeys[0].key.equals(group.readKeys[0].key),
-    'correct readKey'
-  )
+  t.deepEquals(invite.readKeys, group.readKeys, 'correct readKey')
```

Comment:
deepEquals compares array and buffers! Woop!

:fire: I dropped the 'writeKey' requirement in these tests because it's currently unclear what the writeKey should be, AND why would you need a writeKey until AFTER you've accepted an invite. Seems dangerous to return it here